examples/Makefile-nrf52840

```diff
@@ -80,7 +80,7 @@ COMMONCFLAGS                   := \
     $(NULL)
 
 ifeq ($(CERT_LOG),1)
-COMMONCFLAGS += -DOPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT=1
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_LOG_OUTPUT=OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED
```

Comment:
I believe this changes the previous behaviour. When CERT_LOG variable is enabled we expect to use UART as the log output instead of platform defined one (RTT).